website/docs/development/overview.md

```diff
@@ -0,0 +1,29 @@
+---
+id: overview
+title: Overview
+sidebar_label: Overview
+---
+
+This guide assumes you have forked and checked-out the [repository](https://github.com/facebookresearch/hydra).
+It is recommended that you install Hydra in a virtual environment like [conda](https://docs.conda.io/en/latest/) or [virtualenv](https://virtualenv.pypa.io/en/latest/).
+
+## Environment setup
+Install [Miniconda](https://docs.conda.io/en/latest/miniconda.html) and create an empty Conda environment with:
+```
+conda create -n hydra38 python=3.8 -y
+```
+
+<div class="alert alert--info" role="alert">
+<strong>NOTE</strong>:
+The core Hydra framework supports Python 3.6 or newer. You may need to create additional environments for different Python versions if CI detect issues on a supported version of Python.
+</div>
+<br/>
```

Comment:
Docusaurus now have better support for [admonitions](https://v2.docusaurus.io/docs/markdown-features/#calloutsadmonitions).
Try the new syntax.

Reply:
This is sweet!